### DISCUSSION TITLE
How can I turn on the Welcome page when I first install the software?

### DISCUSSION THREAD

**Discussion by khuongviettai** [zed-industries/zed#10894] [Feature Requests]
How can I turn off the old project when I close or close the app?Every time I turn on the app it runs the old project even though I turned it off

**Comment by Glazoff** [comment]
I'm also looking for a solution.

So far I've managed to disable opening the last project. An empty window will open.

Settings  `"restore_on_startup": "none"`

**Comment by matthewamarket** [comment]
@khuongviettai @Glazoff here it is!
[https://github.com/zed-industries/zed/discussions/36532#discussioncomment-14186653](https://github.com/zed-industries/zed/discussions/36532#discussioncomment-14186653)
> Anyway, I found the answer.
> 
> It's available in Help menu > Show Welcome
> 
> Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
